GDJS/Runtime/pixi-renderers/layer-pixi-renderer.ts

```diff
@@ -85,10 +85,10 @@ namespace gdjs {
         // the sprite one and it changes in which direction sprites are rounded.
         // It makes sprites rounding inconsistent with each other
         // and they seems to move on pixel left and right.
-        this._pixiContainer.position.x = Math.round(
+        this._pixiContainer.position.x = Math.ceil(
```

Comment:
Maybe add a link to this PR in the comment so that we can find the context back and why ceil was chosen.